src/suites/cts/format_info.ts

```diff
@@ -0,0 +1,55 @@
+import { poptions } from '../../framework/index.js';
+
+export interface TextureFormatInfo {
+  renderable: boolean;
+  // Add fields as needed
+}
+
+interface TextureFormats {
+  [k: string]: TextureFormatInfo;
+}
+
+// prettier-ignore
+export const textureFormatInfo: TextureFormats = {
+  // Try to keep these manually-formatted in a readable grid.
+  'r8unorm':                { renderable:  true },
```

Comment:
Removing comments such as `// 8-bit formats` make it hard to know why this list is not ordered alphabetically.
How do you feel about keeping those?